.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+    -   id: requirements-txt-fixer
+-   repo: https://github.com/psf/black
+    rev: ''
+    hooks:
+    -   id: black
+-   repo: https://github.com/pycqa/flake8
+    rev: ''  # pick a git hash / tag to point to
```

Comment:
Why you haven't add revision? Project should be ready to run without additional configuration. Please add latest revision.